src/pages/Bets/components/BetModal.tsx

```diff
@@ -4,6 +4,7 @@ import Modal from "../../../components/Modal";
 import RequireWalletButton from "../../../components/RequireWalletButton/RequireWalletButton_View";
 import marketContractJson from "../../../abi/Market.json";
 import { BetHistory } from "../../../types";
+import { Loader } from "../../../components";
```

Comment:
This looks like a stowaway

Reply:
It does, however this is just a little fix to allow it to run, otherwise it errors. Jas raised a separate PR for this (https://github.com/horse-link/horse.link/pull/146) but I feel we can leave it in as it will all get merged

Reply:
Its undercover ;)